Platform/AlderlakeBoardPkg/Library/Stage1BBoardInitLib/MaxLedInit.c

```diff
@@ -0,0 +1,161 @@
+
+#define INTENSITY_REGISTER                    0x02
+#define INTENSITY_MAX_ON_OPERATION            0x3E
+
+#define DECODE_MODE_REGISTER                  0x01
+#define DECODE_MODE_DIGITS_OPERATION          0xFF
+
+
+/**
+  This function does the basic init work for Max6950.
+
+  @param[in]  SpiBaseAddress - Base Address of SPI Controller
+**/
+EFI_STATUS
+EFIAPI
+MaxLedInit (
+  VOID
+  )
+{
+  UINT8  WriBuf[2] = { 0, 0};
+  UINT8  I2cNumber = PcdGet8(PcdI2cIoExpanderPortNumber);
+  UINT8  SlaveAddress = PcdGet8(PcdI2cMaxLedSlaveAddress);
+  UINTN  PciCfgBase;
+  EFI_STATUS Status;
+
+  DEBUG ((DEBUG_INFO, "MaxLedInit Start \n"));
+
+  // I2C Post Code
+  PciCfgBase = (UINTN) SerialIoI2cPciCfgBase (I2cNumber);
+
+  // Set the display_test register fromat command 0x07 to indicate normal operation
+  WriBuf [0] = DISPLAY_TEST_REGISTER;
+  WriBuf [1] = DISPLAY_TEST_NORMAL_OPERATION;
+
+  Status = SerialIoI2cWrite (
+      PciCfgBase,
+      &I2cNumber,
+      SlaveAddress,
+      2,
+      WriBuf,
+      I2C_BUS_TIMEOUT_USEC,
+      I2C_BUS_FREQUENCY_400Khz,
+      NULL,
+      TRUE,
+      TRUE
+  );
+  //
+  // Set config regsiter format command 0x04 to indicate normal operation, slow
+  // blink rate, global blinking function is disabled, global blink timing synch
+  // which blink timing counters are unaffected, global clear digit data which
+  // cause both plane P0 and P1 are unaffected.
+  //
+  WriBuf [0] = CONFIGURATION_REGISTER;
+  WriBuf [1] = CONFIGURATION_NORMAL_OPERATION;
+
+  Status = SerialIoI2cWrite (
+      PciCfgBase,
+      &I2cNumber,
+      SlaveAddress,
+      2,
+      WriBuf,
+      I2C_BUS_TIMEOUT_USEC,
+      I2C_BUS_FREQUENCY_400Khz,
+      NULL,
+      TRUE,
+      TRUE
+  );
+
+  //
+  // Set the scan limit register format command 0x03 to indicate display for digits 3-0
+  //
+  WriBuf [0] = SCAN_LIMIT_REGISTER;
+  WriBuf [1] = SCAN_LIMIT_DIGITS_OPERATION;
+
+  Status = SerialIoI2cWrite (
+      PciCfgBase,
+      &I2cNumber,
+      SlaveAddress,
+      2,
+      WriBuf,
+      I2C_BUS_TIMEOUT_USEC,
+      I2C_BUS_FREQUENCY_400Khz,
+      NULL,
+      TRUE,
+      TRUE
+  );
+
+  //
+  // Set the intensity register format command 0x02 to indicate duty cycle of 63/64 (max on)
+  //
+  WriBuf [0] = INTENSITY_REGISTER;
+  WriBuf [1] = INTENSITY_MAX_ON_OPERATION;
+
+  Status = SerialIoI2cWrite (
+      PciCfgBase,
+      &I2cNumber,
+      SlaveAddress,
+      2,
+      WriBuf,
+      I2C_BUS_TIMEOUT_USEC,
+      I2C_BUS_FREQUENCY_400Khz,
+      NULL,
+      TRUE,
+      TRUE
+  );
+
+  //
+  // Set the decode mode register format command 0x01 to indicate hex decode for digit 7-0
+  //
+  WriBuf [0] = DECODE_MODE_REGISTER;
+  WriBuf [1] = DECODE_MODE_DIGITS_OPERATION;
+
+  Status = SerialIoI2cWrite (
+      PciCfgBase,
+      &I2cNumber,
+      SlaveAddress,
+      2,
+      WriBuf,
+      I2C_BUS_TIMEOUT_USEC,
+      I2C_BUS_FREQUENCY_400Khz,
+      NULL,
+      TRUE,
+      TRUE
+  );
```

Comment:
Following coding style, subsequent lines of multi-line function calls should line up 2 spaces from the beginning of the function name. It looks like:
Status = SerialIoI2cWrite (
                PciCfgBase,
                &I2cNumber,
                ...,
              );

Reply:
updated coding style, done

Reply:
The comment mentioned "line up 2 spaces from the beginning of the function name. ", but the example doesn't show it as expected even I did it in the comment. you could refer   https://github.com/slimbootloader/slimbootloader/blob/45273c82ca7ab9433468d8afc456856ee478ee1f/MdePkg/Library/BasePciCf8Lib/PciCf8Lib.c#L567

Reply:
updated as in the given example